submissions/adeyinkaQuadri/javascript/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="productlist.js"></script>
```

Comment:
It is good coding practice that your src script should be located in the body tag not the head tag